src/api/IZoweTree.ts

```diff
@@ -42,6 +42,13 @@ export interface IZoweTree<T> extends vscode.TreeDataProvider<T> {
      * @param type e.g. zosmf
      */
     addSession(sessionName?: string, type?: string): Promise<void>;
+    /**
+     * Adds a session to the container
+     * @param sessionName
```

Comment:
This JSdoc does not match the type signature of `editSession`. I believe we agreed that we would be removing JSdoc because it duplicates the information from type annotations.

Reply:
ill update it. 

> I believe we agreed that we would be removing JSdoc because it duplicates the information from type annotations.

Then my function will be the only one who doesn't have a comment. It will be inconsistent with others. Maybe a placeholder issue for removing JSDoc in the future can be opened?